src/main/java/org/primefaces/application/resource/csp/Constants.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2009-2018 PrimeTek.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.primefaces.application.resource.csp;
+
+/**
+ * Some constants for Content-Security-Policy headers and directives
+ */
+public enum Constants {
+    
```

Comment:
Sadly I don't prefer Constants as name, give the Enum a real name like CspHeader.java and CspDirective.java.  I find it makes the code more readable and prevents common name like Constants.java

Reply:
@melloware Thanks for your review. 

Okay, I will rename it to `CspHeader` and abbreviate the other class names as well (`Csp` prefix instead of `ContentSecurityPolicy` which is ways too long, almost like Spring class names :wink: ). However there has already been a `Constants` class in PrimeFaces so I thought this might be ok and the package structure is meaningful enough.